File: reimi/frontend/reimi_app/lib/domain/value_objects/weather_personality_axis.dart
import 'package:json_annotation/json_annotation.dart';

enum WeatherPersonalityAxis {
  @JsonValue('SENSITIVITY')
  sensitivity,

  @JsonValue('PREPAREDNESS')
  preparedness,

  @JsonValue('ACTIVITY')
  activity,

  @JsonValue('MOTIVATION')
  motivation,
}

File: reimi/backend/reimi_app/src/main/java/com/reimi/reimi_app/domain/model/weatherpersonality/type/WeatherPersonalityAxis.java
package com.reimi.reimi_app.domain.model.weatherpersonality.type;

public enum WeatherPersonalityAxis {
    SENSITIVITY("感受性"),
    PREPAREDNESS("準備性"),
    ACTIVITY("活動性"),
    MOTIVATION("動機特性");

    private final String label;

    WeatherPersonalityAxis(String label) {
        this.label = label;
    }

    public String getLabel() {
        return label;
    }
}
